lib/Report/Generator/ComponentGenerator.php

```diff
@@ -80,7 +80,7 @@ public function configure(OptionsResolver $options): void
      */
     public function generate(SuiteCollection $collection, Config $config): Reports
     {
-        $dataFrame = $this->transformer->suiteToFrame($collection);
+        $dataFrame = $this->transformer->suiteToFrame($collection, true);
```

Comment:
This isn't related - it's required to show referenced benchmarks when `run` is used instead of `report` (investigate why it  worked with this as `false`)